app/pages/project/stats/charts.cjsx

```diff
@@ -124,7 +124,17 @@ Graph = React.createClass
         if data.index % numberBars
           data.element.attr({style: "display: none"})
     else if data.type == 'bar'
-      data.element.attr({style: "stroke-width: #{100 / length}%"})
+      data.element.attr(
+        style: "stroke-width: #{100 / length}%"
+        focusable: true
```

Comment:
Looks like `focusable` is used by IE and Edge instead of `tabindex` according to http://allyjs.io/data-tables/focusable.html.